src/Command/LoadScreenLayoutsCommand.php

```diff
@@ -26,43 +28,59 @@ public function __construct(
 
     protected function configure(): void
     {
-        $this->addArgument('filename', InputArgument::REQUIRED, 'json file to load');
+        $this->addArgument('filename', InputArgument::REQUIRED, 'json file to load. Can be local file or a URL');
     }
 
     final protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
 
-        if ($filename = $input->getArgument('filename')) {
-            try {
-                $content = json_decode(file_get_contents($filename), false, 512, JSON_THROW_ON_ERROR);
-
-                $screenLayout = new ScreenLayout();
-                $screenLayout->setTitle($content->title);
-                $screenLayout->setGridColumns($content->grid->columns);
-                $screenLayout->setGridRows($content->grid->rows);
-
-                foreach ($content->regions as $localRegion) {
-                    $region = new ScreenLayoutRegions();
-                    $region->setGridArea($localRegion->gridArea);
-                    $region->setTitle($localRegion->title);
-                    $this->entityManager->persist($region);
-                    $screenLayout->addRegion($region);
-                }
+        try {
+            $filename = $input->getArgument('filename');
+            $content = json_decode(file_get_contents($filename), false, 512, JSON_THROW_ON_ERROR);
```

Comment:
Adding a json schema validation to make sure title, grid, etc. are correctly formatted would ensure data integrity.

See:
https://github.com/os2display/display-api-service/blob/develop/src/Command/LoadTemplateCommand.php#L43

Reply:
Agreed. And we have a TODO as to where these schemas are stored and exposed. I like the linked example, but inclining the schema in the command file prevents reuse. 

Afraid it's out of scope right now